prow/oss/plugins.yaml

```diff
@@ -416,3 +397,11 @@ plugins:
     - trigger
     - verify-owners
     - yuks
+
+external_plugins:
+  kubeflow:
+  - name: cherrypicker # Cherry pick PRs to the release branch.
+    events:
+    - issue_comment
+    - pull_request
+    endpoint: http://cherrypicker
```

Comment:
There is no cherrypicker service deployed on this Prow instance so there won't be anything to reach at this endpoint.

Reply:
I see, thanks for checking @cjwagner! I will create separate issue and remove it from this PR.

Reply:
I tested it with https://github.com/kubeflow/manifests/issues/2837#issuecomment-2277454689 and it worked out fine.

Reply:
Amazing, thank you @juliusvonkohout!